features/smoke-tests/users/login.feature

```diff
@@ -3,92 +3,44 @@ Feature: Different types of user can log in
 
 @with-supplier-user
 Scenario: Supplier user can log in
-  Given I visit the homepage
-  And I have an existing supplier user
-  When I click 'Log in'
-  Then I am on the 'Log in to the Digital Marketplace' page
-  When I enter that user.emailAddress in the 'Email address' field
-  And I enter that user.password in the 'Password' field
-  And I click the 'Log in' button
-  Then I see the 'Log out' link
+  Given I have an existing supplier user
+  Then I can log in as that user
```

Comment:
Can we combine all these into a `Scenario Outline` with the role as a param?

Reply:
The only thing that would prevent us doing is having them separately tagged, which I assumed we wanted for a reason.